src/release.py

```diff
@@ -464,8 +464,7 @@ def generate_new_tag(repository, temp_dir, specified_crate):
             versions[crate['crate']] = CRATES_VERSION[crate['crate']]
             if crate['crate'].endswith('-sys') or crate['crate'].endswith('-sys-rs'):
                 version = CRATES_VERSION[crate['crate']]
-    if specified_crate is not None and
-           (specified_crate.endswith('-sys') or specified_crate.endswith('-sys-rs')):
+    if specified_crate is not None and (specified_crate.endswith('-sys') or specified_crate.endswith('-sys-rs')):
```

Comment:
This not was work for me, maybe something with my Python 3.6.4 wrong.

Reply:
Please add parents around instead please. The line is just too long like this. :)

Reply:
You meant this way?

Reply:
I mean like this:

```python
if (specified_crate is not None and
    (specified_crate.endswith('-sys') or specified_crate.endswith('-sys-rs'))):
```

Reply:
You right, that looks better.
Done

Reply:
Python's magic! :p